### DISCUSSION TITLE
MacVim now allows sponsors!

### DISCUSSION THREAD

**Discussion by ychin** [macvim-dev/macvim#1271] [Announcement]
MacVim now allows you to sponsor the team, via GitHub sponsors! If you would like to support MacVim, you can go to https://github.com/sponsors/macvim-dev, and select whichever tier feels comfortable to you.

---

As a background, I have talked to a couple people before on this topic, and there was an issue (#814) filed for adding a sponsorship button as well. I have resisted doing so before, because MacVim development is mostly a volunteer gig for me, and I didn't particularly want to deal with the hassle of setting something like this up. However, MacVim development did start to become more time consuming, and there are also out-of-pocket costs that I end up paying for, such as the Apple Developer Membership program, the macvim.org domain name, and the occasional hardware purchase to test MacVim. Also, it seems that setting up a sponsorship is quite easy these days, and so I felt that it made sense to set up one.

The sponsorship is set up using Open Source Collective on the backend (https://opencollective.com/macvim) and is how the funds will be distributed to team members. You can directly contribute to the collective via that link if you don't have/want a GitHub account.

This is completely optional, of course. So no obligation to do so at all, and thanks for using MacVim!

**Comment by drjackyl** [comment]
Thank you. I just saw it in the latest update-notes and made use of it. MacVim is my default-editor and I appreciate an option to contribute.
